src/lib/blueprint.ts

```diff
@@ -1,15 +1,90 @@
 import type { Openapi } from './openapi.js'
 
+interface Parameter {
+  name: string
+  isRequired: boolean
```

Comment:
We should be able to read these from the openapi spec via https://github.com/seamapi/nextlove/pull/133/files


```suggestion
  isRequired: boolean
  isDeprecated: boolean
  deprecationMessage: string
```